src/minitest/expectations.cr

```diff
@@ -0,0 +1,151 @@
+require "./assertions"
+
+module Minitest
+  # TODO: must_raise / must_raise(class_name)
+  module Expectations
```

Comment:
There may be a way to simplify this with code generation instead of manually duplicating all assertion methods. The problem of writing a macro is that it will require an arbitrary number of arguments and that the following method won't compile (obviously):

```crystal
def new_name(*args, file = __FILE__, line = __LINE__)
end
```

Maybe we should specify the args?

```crystal
macro infect_an_assertion(name, new_name, args = [] of String, dont_flip = false)
  def {{ new_name.id }}({{ args.map(&:id).join(", ").id }}, message = nil, file = __FILE__, line = __LINE__)
  end
end

infect_an_assertion :assert_equal, :must_equal, [:expected]
```